doc/WEATHER_TYPE.md

```diff
@@ -142,19 +120,38 @@ Things that weather can cause to happen.
       ],
       "spawns":
       [{
-        "max_radius":10,
-        "min_radius":10,
-        "target":"mon_zombie_survivor_elite",
-        "hallucination_count":1,
-        "real_count":0
-}]
-```  
-   ### spawn_type
-
-How many and what spawns
-
-##Fields     
-        
+        "max_radius": 10,
+        "min_radius": 10,
+        "target": "mon_zombie_survivor_elite",
+        "hallucination_count": 1,
+        "real_count": 0
+      }]
+}
+```
+
+### `requirements` properties
+
+|     Identifier                 |                              Description                              |
+| ------------------------------ | --------------------------------------------------------------------- |
+| `pressure_min`                 | |
```

Comment:
```suggestion
| `pressure_min`                 | These are all minimum and maximum values for which the weather will occur.  I.e., it will only rain if it is sufficiently humid. |
```